crates/katana/rpc/rpc-types/src/error/starknet.rs

```diff
@@ -118,6 +120,7 @@ impl StarknetApiError {
             StarknetApiError::UnsupportedContractClassVersion => 62,
             StarknetApiError::UnexpectedError { .. } => 63,
             StarknetApiError::ProofLimitExceeded => 10000,
+            StarknetApiError::ChunkSizeTooBig { .. } => 10001,
```

Comment:
We might stick to the spec with `PageSizeTooBig`?
https://github.com/xJonathanLEI/starknet-rs/blob/ee9a5cdd678a3f1941599136432a1502ef772c00/starknet-core/src/types/codegen.rs#L1661

Or @kariy wdyt, are they semantically different?